deployment/terraform/iam.tf

```diff
@@ -36,6 +36,31 @@ data "aws_iam_policy_document" "container_instance_ses_send_email" {
   }
 }
 
+data "aws_iam_policy_document" "batch_manage_jobs" {
+  statement {
+    effect = "Allow"
+
+    resources = ["*"]
+    actions = [
+      "batch:CancelJob",
+      "batch:DescribeJobDefinitions",
+      "batch:DescribeJobQueues",
+      "batch:DescribeJobs",
+      "batch:ListJobs",
+      "batch:SubmitJob",
+      "batch:TerminateJob"
+    ]
+  }
+}
+
+#
+# Custom policies
+#
+resource "aws_iam_policy" "batch_manage_jobs" {
+  name = "BatchManageJobs"
```

Comment:
We'll ultimately want this policy name to be scoped by environment, otherwise Terraform can run into issues when the resource for two managed infrastructures exists within the same AWS account.